src/utils/sentry.ts

```diff
@@ -20,8 +20,8 @@ const sentryOptions = {
     "Unexpected token 'else'",
     'Let/Const 다시 선언',
     'ReferenceError: Let/Const 다시 선언',
-    'TypeError: 특정 호스트 이름을 가진\n' + '서버를 찾을 수 없습니다.',
-    '특정 호스트 이름을 가진\n' + '서버를 찾을 수 없습니다.\n',
+    `TypeError: 특정 호스트 이름을 가진
+    서버를 찾을 수 없습니다.`,
```

Comment:
두 번째 항목이 사라졌어요. 그리고 `\n`도 쓰지 말라고 그래요??

```suggestion
    '특정 호스트 이름을 가진\n서버를 찾을 수 없습니다.',
```